src/main/java/uk/gov/ons/ctp/response/notify/service/impl/NotifyServiceImpl.java

```diff
@@ -244,17 +249,82 @@ private String getTemplateIdByClassifiers(final ActionRequest actionRequest) thr
             classifierMap.put(REGION_CODE, region);
         }
 
-        if (REMINDER_EMAIL.equalsIgnoreCase(actionRequest.getActionType())) {
-            List<String> isReminder = new ArrayList<>();
-            isReminder.add(REMINDER);
-            classifierMap.put(COMMUNICATION_TYPE, isReminder);
-        }
-        if (NOTIFICATION_EMAIL.equalsIgnoreCase(actionRequest.getActionType())) {
-            List<String> isNotification = new ArrayList<>();
-            isNotification.add(NOTIFICATION);
-            classifierMap.put(COMMUNICATION_TYPE, isNotification);
+        NotificationType notificationType =
+                NotificationType.getNotificationTypeFromActionType(actionRequest.getActionType());
+
+        switch (notificationType) {
+            case Notification:
+                List<String> isNotification = new ArrayList<>();
+                isNotification.add(NOTIFICATION);
+                classifierMap.put(COMMUNICATION_TYPE, isNotification);
+                break;
+            case Reminder:
+                List<String> isReminder = new ArrayList<>();
+                isReminder.add(REMINDER);
+                classifierMap.put(COMMUNICATION_TYPE, isReminder);
+                break;
+            default:
+                // This shouldn't happen but if it does, add no classifiers
+                break;
         }
 
         return classifierMap;
     }
+
+    /**
+     * This enum represents the two possible notification types: notifications and reminders.  It was decided in
```

Comment:
The data types for the reminder/notification emails are supposed to be static in the database, this seems like a lot of logic rather than the previous matching on actionType="BSNE" for notifications and actionType="BSRE" for reminders. This is still going to be dependent on the data remaining in the same format, it seems to add complexity for no extra functionality?